---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: 'GLOSSary: The GLobal Ocean 16S subunit web accessible resource'
subtitle: ''
summary: ''
authors:
- M. Tangherlini
- M. Miralto
- C. Colantuono
- M. Sangiovanni
- A. Dell' Anno
- C. Corinaldesi
- R. Danovaro
- M. L. Chiusano
tags:
- '"16S rRNA"'
- '"Marine metagenomics"'
- '"Tara Ocean project"'
- '"Taxonomic analysis"'
categories: []
date: '2018-01-01'
lastmod: 2020-10-16T11:46:04+02:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2020-10-16T09:46:03.413564Z'
publication_types:
- '2'
abstract: 'Background: Environmental metagenomics is a challenging approach that is
  exponentially spreading in the scientific community to investigate taxonomic diversity
  and possible functions of the biological components. The massive amount of sequence
  data produced, often endowed with rich environmental metadata, needs suitable computational
  tools to fully explore the embedded information. Bioinformatics plays a key role
  in providing methodologies to manage, process and mine molecular data, integrated
  with environmental metagenomics collections. One such relevant example is represented
  by the Tara Ocean Project. Results: We considered the Tara 16S miTAGs released by
  the consortium, representing raw sequences from a shotgun metagenomics approach
  with similarities to 16S rRNA genes. We generated assembled 16S rDNA sequences,
  which were classified according to their lengths, the possible presence of chimeric
  reads, the putative taxonomic affiliation. The dataset was included in GLOSSary
  (the GLobal Ocean 16S Subunit web accessible resource), a bioinformatics platform
  to organize environmental metagenomics data. The aims of this work were: i) to present
  alternative computational approaches to manage challenging metagenomics data; ii)
  to set up user friendly web-based platforms to allow the integration of environmental
  metagenomics sequences and of the associated metadata; iii) to implement an appropriate
  bioinformatics platform supporting the analysis of 16S rDNA sequences exploiting
  reference datasets, such as the SILVA database. We organized the data in a next-generation
  NoSQL \"schema-less\" database, allowing flexible organization of large amounts
  of data and supporting native geospatial queries. A web interface was developed
  to permit an interactive exploration and a visual geographical localization of the
  data, either raw miTAG reads or 16S contigs, from our processing pipeline. Information
  on unassembled sequences is also available. The taxonomic affiliations of contigs
  and miTAGs, and the spatial distribution of the sampling sites and their associated
  sequence libraries, as they are contained in the Tara metadata, can be explored
  by a query interface, which allows both textual and visual investigations. In addition,
  all the sequence data were made available for a dedicated BLAST-based web application
  alongside the SILVA collection. Conclusions: GLOSSary provides an expandable bioinformatics
  environment, able to support the scientific community in current and forthcoming
  environmental metagenomics analyses.'
publication: '*BMC Bioinformatics*'
doi: 10.1186/s12859-018-2423-8
---
