---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: 'Assessing viral taxonomic composition in benthic marine ecosystems: Reliability
  and efficiency of different bioinformatic tools for viral metagenomic analyses'
subtitle: ''
summary: ''
authors:
- M. Tangherlini
- A. Dell'Anno
- L. Zeigler Allen
- G. Riccioni
- C. Corinaldesi
tags: []
categories: []
date: '2016-01-01'
lastmod: 2020-10-16T11:46:36+02:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2020-10-16T09:46:34.399889Z'
publication_types:
- '2'
abstract: In benthic deep-sea ecosystems, which represent the largest biome on Earth,
  viruses have a recognised key ecological role, but their diversity is still largely
  unknown. Identifying the taxonomic composition of viruses is crucial for understanding
  virus-host interactions, their role in food web functioning and evolutionary processes.
  Here, we compared the performance of various bioinformatic tools (BLAST, MG-RAST,
  NBC, VMGAP, MetaVir, VIROME) for analysing the viral taxonomic composition in simulated
  viromes and viral metagenomes from different benthic deep-sea ecosystems. The analyses
  of simulated viromes indicate that all the BLAST tools, followed by MetaVir and
  VMGAP, are more reliable in the affiliation of viral sequences and strains. When
  analysing the environmental viromes, tBLASTx, MetaVir, VMGAP and VIROME showed a
  similar efficiency of sequence annotation; however, MetaVir and tBLASTx identified
  a higher number of viral strains. These latter tools also identified a wider range
  of viral families than the others, providing a wider view of viral taxonomic diversity
  in benthic deep-sea ecosystems. Our findings highlight strengths and weaknesses
  of available bioinformatic tools for investigating the taxonomic diversity of viruses
  in benthic ecosystems in order to improve our comprehension of viral diversity in
  the oceans and its relationships with host diversity and ecosystem functioning.
publication: '*Scientific Reports*'
doi: 10.1038/srep28428
---
